variables.tf

```diff
@@ -2259,12 +2259,58 @@ variable "elasticache_outbound_acl_rules" {
   ]
 }
 
+variable "manage_default_security_group" {
+  description = "Should be true to adopt and manage default security group"
+  type        = bool
+  default     = false
+}
+
+variable "default_security_group_name" {
+  description = "Name to be used on the default security group"
+  type        = string
+  default     = ""
+}
+
+variable "default_security_group_ingress" {
+  description = "List of maps of ingress rules to set on the default security group"
+  type        = list(map(string))
+
+  default = [
```

Comment:
Maybe defaults should be empty, because "When Terraform first adopts the Default Security Group, it immediately removes all ingress and egress rules in the Security Group." (from https://www.terraform.io/docs/providers/aws/r/default_security_group.html )

And, if I understand correctly, Terraform won't put original values back, if the user switches off `manage_default_security_group` afterwards.

Previously, there were attempts to manage this resource in this module, but it was a bit too advanced for some use-cases and Terraform 0.11 which we had back then.

Reply:
good point - I think you're right, might be better to use `null` here as it seems to be the safest route usually